src/app/modules/item/pages/item-progress/item-progress.component.html

```diff
@@ -1,5 +1,5 @@
-<ng-container *ngIf="session$ | async as session">
-  <alg-section icon="fa fa-list" i18n-label label="{{session.watchedGroup ? 'Group' : 'User'}} situation">
+<ng-container *ngIf="watchedGroup$ | async as watchedGroup">
+  <alg-section icon="fa fa-list" i18n-label label="Group situation">
```

Comment:
So it never display "User situation" anymore?
Actually, these changes will have severe conflict with #427 that Aleksandr is working on.

Reply:
Fixed